### PR TITLE
Update scenarios.md with Eval 5 baseline performance

### PR DESCRIPTION
**Pull request by yusong-tan**

### CODE REVIEW

File: docs/scenarios.md [hunk 1]
```diff
@@ -277,12 +278,21 @@ may also be loaded by the model.
...
 | PGD (multiple channels)* | yes      | snr=30dB       | 0.13                    | 0.92                         | N/A                   | N/A                        | 100       |
 | PGD (multiple channels)* | yes      | snr=40dB       | 0.13                    | 0.75                         | N/A                   | N/A                        | 100       |
-
-* \*Target attack, where a random target phrase of similar length as the ground truth, was applied but WER wrt the ground truth was calculated
+* \*Targeted attack, where a random target phrase of similar length as the ground truth, was applied but WER wrt the ground truth was calculated
 
 Find reference baseline configurations [here](https://github.com/twosixlabs/armory/tree/8eb10ac43bf4382d69625d8cef8a3e8cb23d0318/scenario_configs)
 * Missing defended baseline is due to current incompatibility of the attack and defense.
 
+Table 2
```
[the remark this answers is not in the document]
Comment (lcadalzo) [reply docs/scenarios.md:286]: For 20dB `scenario_configs/eval5/asr_librispeech/defended_entailment.json`, I'm seeing:

```
"adversarial_total_entailment": [
            {
                "contradiction": 52,
                "entailment": 42,
                "neutral": 6
            }
        ],
        "adversarial_total_word_error_rate": [
            [
                0.3539736713798147,
                [
                    726,
                    2051
                ]
            ]
        ],

```

But the doc lists `0.02/0.96/0.02` for `Entailment/Contradiction/Neutral Rates (Adversarial Defended)`, so this also seems like a major discrepancy

File: docs/scenarios.md [hunk 2]
```diff
@@ -474,30 +483,31 @@ which contains a green-screen intended for adversarial patch insertion. The data
...
+|-------------------|-------------|-----------------------------|----------------------------------|---------------------------------|-----------------------------|------------------|----------------------------------|--------------------------------------|-------------------------------------|---------------------------------|-----------|
+| Robust DPatch     | 0.76/0.72   | 0.19/0.22                   | 3.97/3.48                        | 0.06/0.06                       | 0.75/0.71                   | 0.68/0.66        | 0.27/0.28                        | 4.48/3.65                            | 0.06/0.07                           | 0.67/0.65                       | 31        |
+| Adversarial Patch | 0.76/0.72   | 0.19/0.22                   | 3.97/3.48                        | 0.06/0.06                       | 0.75/0.71                   | 0.54/*           | 0.32/*                           | 22.16/*                              | 0.05/*                              | 0.62/*                          | 31        |
```
Comment (lcadalzo) [inline docs/scenarios.md:495]: For `carla_obj_det_adversarialpatch_undefended.json`, I'm seeing:

```
"adversarial_carla_od_AP_per_class": [
            {
                "class": {
                    "1": 0.43,
                    "2": 0.41,
                    "3": 0.6
                },
                "mean": 0.48
            }
        ],
```

But the doc lists `0.54` for `Adversarial mAP`. This is a less drastic discrepancy, but perhaps nontrivial. The more granular metrics align closely with my results, as do the benign results

File: docs/scenarios.md [hunk 3]
```diff
@@ -474,30 +483,31 @@ which contains a green-screen intended for adversarial patch insertion. The data
...
+| Attack            | Benign  mAP | Benign  Disappearance  Rate | Benign  Hallucination  per Image | Benign  Misclassification  Rate | Benign  True Positive  Rate | Adversarial  mAP | Adversarial  Disappearance  Rate | Adversarial Hallucination  per Image | Adversarial Misclassification  Rate | Adversarial True Positive  Rate | Test Size |
+|-------------------|-------------|-----------------------------|----------------------------------|---------------------------------|-----------------------------|------------------|----------------------------------|--------------------------------------|-------------------------------------|---------------------------------|-----------|
+| Robust DPatch     | 0.76/0.72   | 0.19/0.22                   | 3.97/3.48                        | 0.06/0.06                       | 0.75/0.71                   | 0.68/0.66        | 0.27/0.28                        | 4.48/3.65                            | 0.06/0.07                           | 0.67/0.65                       | 31        |
```
Comment (lcadalzo) [inline docs/scenarios.md:494]: My results for Robust DPatch, undefended and defended, align pretty closely

File: docs/scenarios.md [hunk 4]
```diff
@@ -509,23 +519,26 @@ which contains a green-screen in all frames intended for adversarial patch inser
...
-* **Baseline Model Performance: (results obtained using Armory v0.14.2 and [test data](https://github.com/twosixlabs/armory/blob/r0.14.2/armory/data/adversarial/carla_video_tracking_test.py))**
+  * [Custom Adversarial Texture with Input-Dependent Transformation](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/art_experimental/attacks/carla_adversarial_texture.py)
+* **Baseline Defense**: [Video Compression](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/art_experimental/defences/video_compression_normalized.py)
+* **Baseline Model Performance: (results obtained using Armory v0.15.1 and [dev data](https://github.com/twosixlabs/armory/blob/v0.15.1/armory/data/adversarial/carla_video_tracking_dev.py))**
+
+| Benign Mean IoU | Benign Mean Success Rate | Adversarial Mean IoU | Adversarial Mean Success Rate | Test Size |
+|-----------------|--------------------------|----------------------|-------------------------------|-----------|
+| 0.55/0.57       | 0.57/0.60                | 0.14/0.19            | 0.15/0.20                     | 20        |
```
Comment (lcadalzo) [inline docs/scenarios.md:537]: I should have results for video tracking tomorrow AM

File: docs/scenarios.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (yusong-tan) [reply docs/scenarios.md:286]: @lcadalzo I wasn't clear in the docs. The baseline configuration are meant to be representative and do not contain the optimal attack parameters. To obtain my results, I used `eps_step = 0.05, 0.03, 0.01` for SNR = 20dB, 30dB, and 40dB, respectively.

File: docs/scenarios.md (hunk 2 above)
Comment (yusong-tan) [reply docs/scenarios.md:495]: @lcadalzo These are the `learning_rate` I used to obtain my results, for both single and multimodality, defended and undefended.

`Adversarial Patch`: `learning_rate = 0.003`
`Robust DPatch`: `learning_rate = 0.002`

File: docs/scenarios.md (hunk 4 above)
Comment (yusong-tan) [reply docs/scenarios.md:537]: @lcadalzo I used the default attack hyperparameters for video.

File: docs/scenarios.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lcadalzo) [reply docs/scenarios.md:286]: I also think it makes sense to update the baseline config to the optimal attack parameters, at least for the dB specified in the baseline

**Comment by yusong-tan** [comment]
@lcadalzo Updated the relevant tables with attack parameters used to generate the results.

**Comment by lcadalzo** [comment]
thanks @yusong-tan. Is there any significance of the attack parameter values that exist in the `v0.15.1` config files? I went ahead and updated the config files accordingly in bc50613

File: docs/scenarios.md (hunk 4 above)
Comment (lcadalzo) [reply docs/scenarios.md]: For undefended:
```
adversarial_mean_video_tracking_mean_iou on adversarial examples w.r.t. ground truth labels: 19.88%
```

And defended:
```
adversarial_mean_video_tracking_mean_iou on adversarial examples w.r.t. ground truth labels: 22.19%
```

This differs a bit from the reported `0.14` and `0.19` respectively. I used the default `max_iter: 100` and `step_size: 0.02`

**Comment by yusong-tan** [comment]
@lcadalzo The default attack parameters in `v0.15.1` generally use larger `attack steps` which may prove more effective with smaller `attack iterations`, which we've seen some performers use to iterate defense research faster. My worry with smaller `attack steps` is convergence if someone reduces `attack iterations` independently. This is especially true with the ASR scenarios.

File: docs/scenarios.md (hunk 4 above)
Comment (lcadalzo) [reply docs/scenarios.md]: makes sense, I actually made this change in 15f477c

File: docs/scenarios.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lcadalzo) [reply docs/scenarios.md:286]: I'm able to reproduce your results after aligning our attack parameters

**Pull request merged**